Include required VS version to use NuGet symbol server (#9988)
dotnet/docs: docs/standard/library-guidance/nuget.md

> [!IMPORTANT]
> The NuGet.org symbol server only supports the new [portable symbol files](https://github.com/dotnet/core/blob/master/Documentation/diagnostics/portable_pdb.md) (`*.pdb`) created by SDK-style projects.
>

> > Visual Studio 2017 15.9 or later is required by users of your library to use the NuGet.org symbol server when debugging.

Reviewer: Does it matter if it's my library or someone else's library? Left a suggestion to improve the text.

```suggestion
> To use the NuGet.org symbol server when debugging, you must have Visual Studio 2017 15.9 or later.
```
